### PR TITLE
Symfony 6

### PR DESCRIPTION
**Pull request by ychadwick**
I have made a couple of very changes so that the bundle can be used with symfony 6.

It would be great if you could merge these commits.

Thank you.

### CODE REVIEW

**Review by deguif — APPROVED** [review]

**Comment by deguif** [comment]
@ychadwick I just fixed the CI on master branch. Could you rebase your branch so that CI can be executed now with your modifications

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Versions and dependencies
 | [**2.0**] (*unmaintained*) | ^2.0      | ~2.0.0                   |                |
 | [**1.x**] (*unmaintained*) | ^1.0      | ^2.0                     |                |
 
+
```
Comment (deguif) [inline README.md:30]: This extra line added can probably be removed?

**Comment by ychadwick** [comment]
Ok, I am not sure why the coding style change failed, but otherwise it all looks good

File: Command/InstallFontCommand.php [hunk 2]
```diff
@@ -19,10 +19,12 @@
 use Symfony\Component\Filesystem\Filesystem;
 use Symfony\Component\Finder\Finder;
 use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
```
Comment (deguif) [inline Command/InstallFontCommand.php:22]: Should be alpha sorted, that's why CS checks are failing

File: README.md (hunk 1 above)
Comment (ychadwick) [reply README.md:30]: Sure Line is removed

File: Command/InstallFontCommand.php (hunk 2 above)
Comment (ychadwick) [reply Command/InstallFontCommand.php]: Changed the sort order, should work now

**Comment by deguif** [comment]
Thanks @ychadwick

**Pull request merged**